packages/terra-table/tests/nightwatch/multi-select-table/multi-select-table-spec.js

```diff
@@ -0,0 +1,286 @@
+/* eslint-disable no-unused-expressions */
+// eslint-disable-next-line import/no-extraneous-dependencies
+const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
+
+/**
+ * Create NightWatch expects against the row element at the 0-indexed position.
+ */
+function expectRowForBrowserAtIndex(browser, index) {
+  return browser.expect.element(`[class*="row"]:nth-child(${index + 1})`);
+}
+
+/**
```

Comment:
These helper functions are a good idea!! 

Lets update the names to be a bit more readable, maybe something like: `expectRowIsSelected` and `expectRowIsNotSelected`. If anything, this needs to be updated from the use of the term 'assert'. The assert and expect syntax in nightwatch are different and our team is in the process of moving to use only expect. Plus, assert is misleading given these helpers actually do use expect by calling `expectRowForBrowswerAtIndex`.

Reply:
Thanks! =) I left the original functions with ForBrowserAtIndex for clarity but I adjusted them all to use expect and for the functions used in the actual tests, I removed AtIndex as per your suggestion.